content/javascript/concepts/dates/terms/getTime/getTime.md

```diff
@@ -0,0 +1,28 @@
+---
+Title: '.getTime()' 
+Description: 'Returns the number of milliseconds since 1st January 1970' 
+Subjects: 
+  - 'Web Development'
+Tags: 
+  - 'Date'
+  - 'Methods'
+CatalogContent: 
+  - 'path/web-development'
+---
+
+The **`getTime()`** method returns the number of milliseconds since the 1st of January 1970.
+
+## Syntax
+
+```js
+myDate.getTime()
+```
+
+The returned value from `getTime()` is an number value showing the number of milliseconds since the 1st January 1970.
+
+## Example
+
```

Comment:
Once you edit the example to do something with the result of the function, add a sentence introducing the example and explaining what it does.